config/routes.rb

```diff
@@ -10,7 +10,7 @@
   get '/login', to: 'sessions#new'
   post '/login', to: 'sessions#create'
 
-  delete '/logout', to: 'sessions#destroy'
+  get '/logout', to: 'sessions#destroy'
```

Comment:
So I was still having the problem where the logout route went to /categories and it would error out. Changing the order of the routes themselves didn't seem to help for me. But changing it to a GET and taking the delete out of the logout method did. I think having it as a GET is still okay since it's a separate thing from /login... I know we're not really "getting" a session though. If there's a better way to do it I'm all for it.